automatic/datagrip/tools/ChocolateyInstall.ps1

```diff
@@ -1,13 +1,19 @@
 ﻿$ErrorActionPreference  = 'Stop'
 
+$toolsDir     = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+# Workaround for https://youtrack.jetbrains.com/issue/IDEA-202935
+$programfiles = (${env:ProgramFiles(x86)}, ${env:ProgramFiles} -ne $null)[0]
+New-Item -ItemType Directory -Force -Path $programfiles\JetBrains
```

Comment:
Any reason why you're creating the directory manually here (is this perhaps the workaround you've mentioend)?

Still, still would be better to provide as a package parameter, in cases where users wishes to
change the default installation directory.
Instead of blindly expecting the program to always be installed to programs directory (unless the directory doesn't change when using the `/D=` argument).

Reply:
Hi @AdmiringWorm, yes this is the default installation folder. It could be overwritten with the `/D=` parameter. This should be fixed in 2018.3.1 and up.

How would you like to procede?

Reply:
I would like to see something similiar to what we provide in the openssl.light package:
https://github.com/chocolatey/chocolatey-coreteampackages/blob/8bd34b2f74aee02cb7442db7c7c68034366c800c/automatic/openssl.light/tools/chocolateyInstall.ps1#L5-L7

This would require an update to the Readme.md file as well though:
See the package parameter section in the following example:
https://github.com/chocolatey/chocolatey-coreteampackages/blob/8bd34b2f74aee02cb7442db7c7c68034366c800c/automatic/openssl.light/Readme.md#package-parameters

If you could add this, I have no issues with merging this PR.